deploy/eck-stack/charts/eck-fleet-server/values.yaml

```diff
@@ -79,12 +79,14 @@ policyID: eck-fleet-server
 # No default is currently set, refer to https://github.com/elastic/cloud-on-k8s/issues/7429.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elastic-agent-fleet-configuration.html#k8s-elastic-agent-fleet-configuration-chose-the-deployment-model
 #
-# deployment:
-#   replicas: 1
-#   podTemplate:
-#     spec:
-#       serviceAccountName: fleet-server
-#       automountServiceAccountToken: true
+deployment:
+  replicas: 1
+  podTemplate:
+    spec:
+      serviceAccountName: fleet-server
+      automountServiceAccountToken: true
+      securityContext:
+        runAsUser: 0
```

Comment:
I've been specifically avoiding setting these defaults because of a helm issue we encountered in the past: https://github.com/elastic/cloud-on-k8s/issues/7429
https://github.com/helm/helm/issues/12469#issuecomment-1755986299

Reply:
As discussed I am going to remove them and we need to add a note to our release notes about this breaking change.